<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtFunc</class>
 <widget class="QWidget" name="RtFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>68</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>68</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>输出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>335</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="btnCos">
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnSin">
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnTan">
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnCot">
      <property name="text">
       <string>cot</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnSec">
      <property name="text">
       <string>sec</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnCsc">
      <property name="text">
       <string>csc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
